src/test/java/com/bobocode/svydovets/source/autowire/constructor/ConstructorInjectionServiceWithoutAutowire.java

```diff
@@ -0,0 +1,30 @@
+package com.bobocode.svydovets.source.autowire.constructor;
+
+
+import svydovets.core.annotation.Autowired;
+import svydovets.core.annotation.Component;
+
+@Component
+public class ConstructorInjectionServiceWithoutAutowire {
```

Comment:
The last one :) Please, rename it to "**Invalid**ConstructorInjectionServiceWithAutowire" or smth like that